### PR TITLE
Set tests for wbn-sign package

### PR DESCRIPTION
**Pull request by sonkkeli**

### CODE REVIEW

File: js/sign/tests/integrityblock_test.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    const keypair = crypto.generateKeyPairSync('ed25519');
+    const file = path.resolve(__dirname, 'testdata/unsigned.wbn');
+    const contents = fs.readFileSync(file);
+    const signer = new wbnSign.IntegrityBlockSigner(contents, {
+      key: keypair.privateKey,
+    });
+    return signer;
+  }
+
+  it('Only ed25519 type of key is accepted.', () => {
```
Comment (irori) [inline js/sign/tests/integrityblock_test.js:19]: Test description should follow the [BDD](https://en.wikipedia.org/wiki/Behavior-driven_development) style where method name + description read as an English sentence. e.g.
```javascript
it('accepts only ed25519 type of key.' ...)
```

File: js/sign/tests/integrityblock_test.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    const signer = initSignerWithTestWebBundleAndKeys();
+    const integrityBlock = signer.sign();
```
Comment (irori) [inline js/sign/tests/integrityblock_test.js:21]: This test should also verify that a non-ed25519 key is rejected.

**Comment by sonkkeli** [comment]
PTAL :)

**Comment by irori** [comment]
> PTAL :)

Did you push the new commit?

**Comment by sonkkeli** [comment]
Ooops, now I did

**Review by irori — APPROVED** [review]
lgtm

**Pull request merged**